jetson/db/data_insertion.py

```diff
@@ -0,0 +1,24 @@
+from ftp_transfer import ftp_transfer
+from db_connection import sql_insert, IMAGE, BBOX
+
+
+def data_insert(image_name, image_date, image_time, init_vec, bboxes, input_dir):
+    """Transfer image to remote storage path using SFTP, and inserts image

```

Comment:
Can you clean up this description. The purpose of the description is to inform the user or developer of what this function does not how it does it. So you don't need to mention the type of database or protocols used that can be defined in a note after the description.

example:

```
"""
Transfers image to remote storage then saves metadata and bounding box data in database
"""
```